scripts/ci/lib.sh

```diff
@@ -344,6 +344,21 @@ push_race_condition_debug_image() {
     oc_image_mirror "$MAIN_RCD_IMAGE" "${registry}/main:$(make --quiet tag)-rcd"
 }
 
+push_mock_grpc_server_image() {
+    image="${registry}/grpc-server:$(make --quiet tag)"
```

Comment:
`${registry}` will not be defined at this point.

Reply:
It works. It doesn't crash. E.g. I have the following output 

INFO: Tue Oct  4 14:02:07 UTC 2022: Pushing the mock grpc server image: registry.build01.ci.openshift.org/ci-op-9qs70v1n/pipeline@sha256:6048c035ad7b908872f46889ab0d4e96f28ba9e651660b1b80d2f3882b275899 to quay.io/rhacs-eng/grpc-server:3.72.x-208-g8ef416ffb1

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/32814/rehearse-32814-pull-ci-stackrox-stackrox-jv-rox-12933-build-and-push-mock-grpc-server-in-osci-push-images/1577291009178472448/artifacts/push-images/push/build-log.txt

Reply:
I am not sure where it is being set though.